packages/website/docs/docs/side-menu.md

```diff
@@ -25,40 +25,38 @@ You can also click the drag handle in the Block Side Menu (`⠿`) to open the Dr
 
 ## Custom Drag Handle Menu
 
-BlockNote lets you customize which items appear in the Drag Handle Menu. Have a look at the example below, in which the color picker item is replaced with a custom item that opens an alert.
+If you want to change the items in the Drag Handle Menu, or replace it altogether, you can do that using a React component.
```

Comment:
this document is confusing. We don't explain how to customize the sidemenu, but the draghandle menu instead. agree?

I think maybe we should first explain how to customize the side menu, and only then how to change the drag menu